benefits/eligibility/templates/eligibility/includes/selection-label--senior.html

```diff
@@ -0,0 +1,24 @@
+{% extends "eligibility/includes/selection-label.html" %}
+{% load i18n %}
+
+{% block label %}
+    {% translate "eligibility.pages.index.senior.label" %}
+{% endblock label %}
+
+{% block description %}
+    {% translate "eligibility.pages.index.senior.description" %}
+
+    <!-- Button to trigger modal -->
+    <button id="login" type="button" class="modal-trigger--link" data-bs-toggle="modal" data-bs-target="#login-gov-help">
+        <span class="fallback-text color-logo">Login.gov</span>
+    </button>
+
+    {% include "eligibility/includes/modal--senior-help.html" with id="login-gov-help" %}
+
+    <script nonce="{{ widget.csp_nonce }}">
+        const modal = document.getElementById("login-gov-help");
+        modal.addEventListener("click", function (event) {
+            event.preventDefault();
```

Comment:
Oof, unfortunately this is breaking the link, which is also processed as a `click` event.

Let's remove this and file an issue to follow-up.